07.ls_object/entry.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'etc'
+
+class Entry
+  attr_reader :name
+
+  FILE_TYPE = { fifo: 'p', characterSpecial: 'c', directory: 'd', blockSpecial: 'b', file: '-', link: 'l', socket: 's' }.freeze
```

Comment:
```suggestion
  FILE_TYPE = {
    fifo: 'p',
    characterSpecial: 'c',
    ....
```

というように改行した方が読みやすくなりそうです

Reply:
修正しました。